code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -804,9 +804,18 @@
 		draw_color ||= (species_color) || (skin_tone && skintone2hex(skin_tone))
 
 	if(draw_color)
-		limb.color = "[draw_color]"
+
+		//SKYRAT EDIT BEGIN - Alpha values on limbs //We check if the limb is attached and if the owner has an alpha value to append
+		if(owner && owner.dna.species)
+			limb.color = "[draw_color][num2hex(owner.dna.species.specific_alpha, 2)]" //honestly this is cursed as hell
```

Comment:
make this into a ternery
`requirement ? passed : failed`
E.G. `aux.color = owner?.dna.species ? "[draw_color][num2hex(owner.dna.species.specific_alpha, 2)]" : "[draw_color]"`

Reply:
Honestly that's fair. Thought it would've been a little easier to read this way.

One second.